internal/plugin/factory.go

```diff
@@ -20,9 +21,17 @@ var exePath string
 
 func init() {
 	var err error
-	exePath, err = os.Executable()
-	if err != nil {
-		panic(err)
+	exp := os.Getenv("WAYPOINT_PLUGIN_PATH")
```

Comment:
Is this something we expect people to generally use or was this for dev only? If its for dev only maybe we should rename this to not the `WAYPOINT_` prefix which makes it look official.

Reply:
I mean, it is useful to be able to control where the builtin plugins are executed from, in case you want to use a wrapper script or something.

Reply:
No problem, it was a genuine question, I didn't know.

In that case, I might recommend naming this `WAYPOINT_BUILTIN_PLUGIN_EXE` or something. I imagine at some point we'll drop the builtin plugins, but sitll have plugins, and `WAYPOINT_PLUGIN_PATH` sounds too general to me.

Reply:
Sure, that sounds good.